test/knex.test.js

```diff
@@ -259,6 +260,77 @@ describe('transaction', function () {
 
     throw new Error('expected the call to throw')
   })
+
+
+  it('adopt-commit', async () => {
+
+    let foo1_id
+
+    const trx = await knex(DbConfigPG).transaction()
+    const txseneca  = await si.entity.adopt({handle:trx})
+
+    //I'm using the same Knex instance....
+    const foosSave = await txseneca.entity('foo').data$({p1:'t1'}).save$()
```

Comment:
take more care to define a set of names to use: foo0, foo1, foo3, etc

Reply:
Got it, named to foo<number>